portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -50,14 +54,13 @@ public Comment(String c, String n){
   public void init() {
         
     comments = new ArrayList<>();
```

Comment:
You shouldn't need this array anymore, since we're loading/saving using DataStore